databasin/client.py

```diff
@@ -149,7 +150,51 @@ def get_temporary_file(self, uuid):
         except HTTPException as e:
             raise_for_authorization(e.response, self.username is not None)
             raise
+    
+    def import_lpk(self, lpk_file):
+        if lpk_file.endswith('.lpk'):
+            f = open(lpk_file, 'a+b')
+        else:
+            raise ValueError('File must be an ArcGIS Layer Package with a .lpk extension')
+
+        filename = '{0}.lpk'.format(os.path.splitext(os.path.basename(lpk_file))[0])
```

Comment:
If `lpk_file` already ends with `.lpk` (as enforced on L155), why not just grab the basename?

```python
filename = os.path.basename(lpk_file)
```